src/Faithlife.Data/DbConnectorCommand.cs

```diff
@@ -474,5 +521,76 @@ private async IAsyncEnumerable<T> DoEnumerateAsync<T>(Func<IDataRecord, T>? map,
 			}
 			while (await methods.NextResultAsync(reader, cancellationToken).ConfigureAwait(false));
 		}
+
+		private sealed class PreparedCommand : IDbCommand
```

Comment:
Ugh, I just realized that this will prevent async methods from being used with prepared commands. I should probably at least derive from `DbCommand`...

Reply:
No, I'll just unwrap. d5a0fb7844644bb73ef18bef2d744cb356807d7d

Reply:
What's the purpose of the `PreparedCommand` class? I'm having difficulty figuring out why it exists.